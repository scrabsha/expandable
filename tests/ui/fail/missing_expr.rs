#![allow(unused)]

#[expandable::expr]
macro_rules! test {
    () => {{
        let a = ;
    }}
}

fn main() {}
