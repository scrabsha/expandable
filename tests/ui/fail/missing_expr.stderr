error: Potentially invalid expansion. Expected an expression, an identifier, `::`, `<`, `Self`, `break` or 13 others.
 --> tests/ui/fail/missing_expr.rs:6:17
  |
6 |         let a = ;
  |                 ^
